docs/LogGuide.md

```diff
@@ -838,47 +848,72 @@ Network ability lines are a combination of raw network data
 and frequently sampled data from memory
 (e.g. the `66480` current hp value and `-653.0394` x position).
 
-This means that there's a number of caveats going on to handling all the data in these lines.  The raw network data is subject to change over time from ff14 servers.  Also, the data from memory may be slightly stale and out of date
+This means that there's a number of caveats going on to handling all the data in these lines. The raw network data is
+subject to change over time from ff14 servers. Also, the data from memory may be slightly stale and out of date
+
+#### Action Effects
+
+Each ability may have one or more effects. These are indicated by the flagsX and damageX fields, between `targetName`
+and `targetCurHp`. There are eight pairs, each with a 'flags' field and a 'damage' field. The damage field is not
+necessarily always damage. For example, for a buff application, it indicates which buff ID is about to be applied.
+
+This means that damage is not necessarily the first flag. If you are simply trying to determine how much damage a
+particular ability did, there is no specific field you can look at - you must parse all of them and find the one that
+indicates damage.
 
 #### Ability Flags
 
-Damage bitmasks:
+The 'flags' field for each pair of values can be further broken down.
```

Comment:
I think you misunderstand how action effects work. There is no `damage` field, it's 4 generic params and a flags field, and the usage of *any* of those fields or flags depends entirely on the category of the effect. There are currently 63 effect categories, not just the 2 you mention.

Reply:
Yes, that's why I corrected this part of the documentation. Calling it "damage" is more or less just a holdover form the old version of it.